controllers/article.js

```diff
@@ -0,0 +1,54 @@
+const Article = require('../models/article');
+const BadRequestError = require('../errors/BadRequestError');
+const ForbiddenError = require('../errors/ForbiddenError');
+const NotFoundError = require('../errors/NotFoundError');
+
+const getAllArticles = (req, res, next) => {
+  const owner = req.user._id;
+  Article.find({ owner })
+    .then((articles) => res.status(200).send(articles))
+    .catch(next);
+};
+
+const createArticle = (req, res, next) => {
+  const { keyword, title, text, date, source, link, image } = req.body;
+  Article.create({
+    keyword,
+    title,
+    text,
+    date,
+    source,
+    link,
+    image,
+    owner: req.user._id,
+  })
+    .then((article) => res.status(201).send({ data: article }))
```

Comment:
In some of the cases you sending with key data but in some of the not, can you make in the same way, or send data or relevant object.